samples/run-series.sh

```diff
@@ -39,8 +39,17 @@ for i in 1 4 16 32 64 80; do
     grep "Total uncompress" $plotfn | grep "threads $i," | awk '{print "uncompress,"$11 $7 $5 }' >> ${run_series_report}
 done
 
-# Stress test for a system checkstop on kernel.
+# The following tests may crash the system on a failure.
 if $unsafe; then
     head -c 1048576 $1 > junk2; # 1 Mb
+    # Stress test for a system checkstop on kernel.
     NX_GZIP_PASTE_RETRIES=200 ./compdecomp_th junk2 400 24
+    # Stress test for disabling IRQ on kernel.
+    echo "Testing for kernel disabling IRQ..."
+    NX_GZIP_TIMEOUT_PGFAULTS=3 ./bad_irq_check junk2 400 96
+    echo "Success!"
+    # Stress test to check if the system handles many page faults.
+    echo "Checking many random page faults..."
+    ./rand_pfault_check junk2 100 480
+    echo "Success!"
 fi
```

Comment:
Everything in this unsafe section are tests that are saved in the samples directory.
Why are we storing tests under samples/?  Should they be stored under test/?

Reply:
They are under samples/ because they depend on compdecomp_th. We could move they to tests/ but IMO it's better not have files in tests/ depending on files on samples/.
With some work on samples we may be able to move compdecomp_th to tests, I'm not sure run-series.sh should be on samples/ but this is in a larger scope than this PR.

Reply:
> but this is in a larger scope than this PR.
Agreed.
Could you create an issue for this so that we don't miss it, please?